packages/@aws-cdk/aws-apigatewayv2/lib/http/route.ts

```diff
@@ -61,7 +61,7 @@ export class HttpRouteKey {
     if (path !== '/' && (!path.startsWith('/') || path.endsWith('/'))) {
       throw new Error('path must always start with a "/" and not end with a "/"');
     }
-    return new HttpRouteKey(`${method ?? 'ANY'} ${path}`, path);
+    return new HttpRouteKey(`${method ?? HttpMethod.ANY} ${path}`, path);
```

Comment:
@nija-at This was hardcoded to `ANY` so I've changed and I've decided to change it to use enum instead. Kinda our fo the scope for this PR but somewhat related. Let me know if that's ok.

Reply:
Yep, that's ok 👍